src/java/main/org/apache/zookeeper/server/persistence/FileTxnLog.java

```diff
@@ -102,6 +102,20 @@
     /** Maximum time we allow for elapsed fsync before WARNing */
     private final static long fsyncWarningThresholdMS;
 
+    /**
+     * This parameter limit the size of each txnlog to a given limit (KB).
+     * It does not affect how often the system will take a snapshot
+     * [zookeeper.snapCount]
+     *
+     * The feature is disabled by default (-1)
+     */
+    public static final String LOG_SIZE_LIMIT = "zookeeper.txnlogSizeLimit";
```

Comment:
I think it would be nice to indicate the unit in the name of the setting. e.g. `txnlogSizeLimitInKb`